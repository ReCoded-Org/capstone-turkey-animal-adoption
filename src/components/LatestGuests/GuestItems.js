import React from "react";
import { Col } from "react-bootstrap";
import "./LatestGuests.css";

const GuestItems = ({ img, buttonText, id }) => {
  const background = {
    backgroundImage: `url(${img})`,
  };

  return (
    <Col lg={4} className="mb-5 guestItems" style={background}>
      <button className="viewButton">{buttonText}</button>
    </Col>
  );
};

export default GuestItems;
