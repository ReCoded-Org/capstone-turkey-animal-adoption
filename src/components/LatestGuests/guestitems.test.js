import React from "react";
import renderer from "react-test-renderer";
import GuestItems from "./GuestItems";

it("renders Guest Items correctly", () => {
  const tree = renderer.create(<GuestItems />).toJSON();
  expect(tree).toMatchSnapshot();
});
