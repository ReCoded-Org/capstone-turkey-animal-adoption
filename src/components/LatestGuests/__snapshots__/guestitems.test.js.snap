// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-5 guestItems col-lg-4"
  style={
    Object {
      "backgroundImage": "url(undefined)",
    }
  }
>
  <button
    className="btn btn-primary"
    disabled={false}
    id="viewButton"
    onClick={[Function]}
    type="button"
  />
</div>
`;
